test/File/RenameUploadTest.php

```diff
@@ -417,7 +417,7 @@ public function testReturnUnfiltered($input)
      * @see https://github.com/zendframework/zend-filter/issues/77
      * @return void
      */
-    public function testCachesResultsOfFilteringSAPIUploads()
+    public function testFilterDoesNotAlterUnkownFileDataAndCachesResultsOfFilteringSAPIUploads()
```

Comment:
The fact that we are testing the cache is important but not the main one. It's important to not alter the other data.